src/IO.Ably.Tests.Shared/ClientOptionsTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IO.Ably.AcceptanceTests;
+using Microsoft.Build.Logging;
+using Xunit;
+
+namespace IO.Ably.Tests.Shared
+{
+    public class ClientOptionsTests
+    {
+        [Fact]
+        [Obsolete]
+        public void Options_WithLogHanderReturns_LogHandlerWithSameLoggerSink()
```

Comment:
I'm not sure what the naming convention here is but wouldn't this be more logical?
```suggestion
        public void Options_WithLogHander_ReturnsLogHandlerWithSameLoggerSink()
```

Reply:
Sure, I guess I was referring to the test naming convention from https://github.com/ably/ably-dotnet/blob/344d1d2741daee45ca37b411c07cfc35c4e88a8b/src/IO.Ably.Tests.Shared/ApiKeyTests.cs#L17.
But surely, makes sense to change it to the above name.